common/expirecontainer.h

```diff
@@ -374,9 +376,9 @@ class __ObjectCache : public ObjectCacheBase {
     using ValEntity = typename Item::ValEntity;
     using Borrow = typename Base::Borrow<__ObjectCache>;
 
-    __ObjectCache(uint64_t expiration) : Base(expiration, expiration / 16) {}
-    __ObjectCache(uint64_t expiration, uint64_t timer_cycle)
-        : Base(expiration, timer_cycle) {}
+    explicit __ObjectCache(uint64_t lifespan) : Base(lifespan, lifespan / 16, true) {}
+    __ObjectCache(uint64_t lifespan, uint64_t timer_cycle, bool expire_if_possible)
```

Comment:
It is better to keep API stable.

two-parameter constructor should not be break, and leaves same action as it used to be.